views/default/js/scheduling/table.js

```diff
@@ -3,169 +3,74 @@ elgg.provide('elgg.scheduling');
 
 /**
  * Add a new column to each of the existing rows
- *
- * @param {Object} event
  * @return void
  */
-elgg.scheduling.addColumn = function(event) {
-	var rows = $('#elgg-table-scheduling tr');
-
-	// Count columns leaving out the first one with dates
-	var columnCount = $('#elgg-table-scheduling tr th').length -1;
-
-	rows.each(function(key, value) {
-		if (key === 0) {
-			$(this).append('<th>' + elgg.echo('scheduling:slot:title', [columnCount]) + '</th>');
-			return true;
+elgg.scheduling.addColumn = function () {
```

Comment:
You haven't converted this into an AMD module, so I assume everything else is also compatible with Elgg 1.8?